docs/sources/configure-client/grafana-agent.md

```diff
@@ -14,95 +14,57 @@ to the Grafana Phlare database.
 
 To be able to pull profiles from applications, your applications needs to expose [pprof endpoints](https://pkg.go.dev/net/http/pprof).
 
-For more information about how to instrument your application with pprof endpoints see the [language sdk documentation]({{< relref "./language-sdks/">}}) section.
+### Prerequisites
 
-By default the agent will pull profiles every 10s with no timeout using `http` scheme. This can be configured in the `scrape_configs` section:
+Before proceeding with the configuration, ensure that you have the following:
 
-```yaml
-scrape_configs:
-  - job_name: 'default'
-    scrape_interval: 10s
-    scrape_timeout: 0s
-    scheme: http
-```
+- Go application with Pyroscope agent instrumentation (more languages coming soon)
+- Docker and Docker Compose installed on your system
 
-If you don't provide an override of `profiling_config` for each `scrape_configs`, the agent will use the following default:
-
-```yaml
-profiling_config:
-  path_prefix: ''
-  pprof_config:
-    memory:
-      enabled: true
-      path: "/debug/pprof/allocs"
-      delta: false
-    block:
-      enabled: true
-      path: "/debug/pprof/block"
-      delta: false
-    goroutine:
-      enabled: true
-      path: "/debug/pprof/goroutine"
-      delta: false
-    mutex:
-      enabled: true
-      path: "/debug/pprof/mutex"
-      delta: false
-    process_cpu:
-      enabled: true
-      path: "/debug/pprof/profile"
-      delta: true
-```
+### Setup
 
-For example if you want to:
-
-- fully disable the `goroutine` profile
-- change the `memory` to a different path
-- add [`fgprof`](https://github.com/felixge/fgprof) mixed I/O and CPU profiles.
-
-You can override the `profiling_config` as follow:
-
-```yaml
-scrape_configs:
-  - job_name: 'default'
-    scrape_interval: 10s
-    profiling_config:
-      pprof_config:
-        goroutine:
-          enabled: false
-        memory:
-          path: "/debug/pprof/heap"
-        fgprof:
-          enabled: true
-          path: "/debug/fgprof"
-          delta: true
-```
+To set up the Grafana Agent in pull mode, follow these steps:
 
-Finally if you're running pprof endpoints below a path prefix, you can specify it in the `profiling_config`:
+Clone the repository or navigate to your project directory.
 
-```yaml
-scrape_configs:
-  - job_name: 'default'
-    scrape_interval: 10s
-    scrape_timeout: 0s
-    scheme: http
-    profiling_config:
-      path_prefix: '/app'
+Create the following directory structure for the example files:
```

Comment:
Wonder if it is better to describe a git clone instead.

We already have a bit of a demo here: https://github.com/grafana/phlare/tree/main/tools/docker-compose

It is "just" without Grafana Agent